api/src/main/java/org/openmrs/Encounter.java

```diff
@@ -353,6 +355,36 @@ public void removeObs(Obs observation) {
 	public void setOrders(Set<Order> orders) {
 		this.orders = orders;
 	}
+
+	/**
+	 * @return Returns the order groups
```

Comment:
Added @since 1.12 to 
: OrderSet, OrderGroup, OrderSetMember classes,
: OrderSetService, HibernateOrderSetDAO, 
: new methods in OrderService
: new fields in OrderClass